pkg/tfgen/generate.go

```diff
@@ -864,6 +864,16 @@ func input(sch *schema.Schema) bool {
 	return sch.Optional || sch.Required
 }
 
+// isComputedOverride checks whether the given property is marked to be a computed only value. This
+// ignores any sign of it being an input value
+func isComputedOverride(info *tfbridge.SchemaInfo) bool {
+	if info.MarkAsComputedOnly != nil {
+		return *info.MarkAsComputedOnly
+	}
+
+	return false
```

Comment:
```suggestion
	return info.MarkAsComputedOnly != nil && *info.MarkAsComputedOnly
```

Reply:
done!

Reply:
Looks like this change didn't actually get made?

Reply:
My bad - I didn't push the change!